lib/katello/plugin.rb

```diff
@@ -165,6 +165,15 @@
        :after => :content_hosts,
        :turbolinks => false
 
+  menu :labs_menu,
+       :experimental_ui,
+       :url => '/xui',
+       :url_hash => {:controller => 'katello/react',
+                     :action => 'index'},
+       :caption => N_('Experimental UI'),
```

Comment:
Will all experimental UI live under this across Foreman and plugins? I only ask cause this could be confusing if its Content specific UI.

Reply:
This would likely just be Katello experimental UI.  I see your point about it being confusing, any ideas what else I could call this?  What about "Katello Experimental UI" or "Content Experimental UI"?